tests/api_connexion/endpoints/test_dag_run_endpoint.py

```diff
@@ -307,6 +336,41 @@ def test_should_respond_200(self, session):
             "total_entries": 2,
         }
 
+    def test_filter_by_state(self, session):
+        self._create_test_dag_run_with_queued()
```

Comment:
```suggestion
        self._create_test_dag_run()
        self._create_test_dag_run(state='queued')
```

Reply:
Hi @ephraimbuddy, understood this but we need to clarify the previous point before going further with this. Thx.